modules/migration-configuring-mcg.adoc

```diff
@@ -11,30 +11,38 @@
 = Retrieving Multicloud Object Gateway credentials
 
 ifdef::installing-3-4,installing-mtc[]
-You must retrieve the Multicloud Object Gateway (MCG) credentials and S3 endpoint in order to configure MCG as a replication repository for the {mtc-full} ({mtc-short}).
+You must retrieve the Multicloud Object Gateway (MCG) credentials and S3 endpoint to configure MCG as a replication repository for the {mtc-full} ({mtc-short})
+
+You must retrieve the Multicloud Object Gateway (MCG) credentials to create a `Secret` custom resource (CR) for {mtc-short}.
```

Comment:
Same thing here. 
```suggestion
You must retrieve the Multicloud Object Gateway (MCG) credentials in order to create a `Secret` custom resource (CR) for {mtc-short}.
```

Reply:
Unresolving this as I see its still needing to be addressed or a reason why not. @RichardHoch I'll let you tell me when you're ready for merge after you address this.

Reply:
@skrthomas You're right, I missed this. Sorry!